lib/bridge.js

```diff
@@ -672,6 +679,48 @@ Bridge.prototype._updateIntents = function(event) {
     });
 };
 
+/**
+ * Returns a PrometheusMetrics instance. The instance will be registered with
+ * the HTTP server so it can serve the "/metrics" page in the usual way.
+ */
+Bridge.prototype.getPrometheusMetrics = function() {
+    if (this._metrics) return this._metrics;
+
+    var metrics = this._metrics = new PrometheusMetrics();
+
+    metrics.registerMatrixSdkMetrics();
+
+    // TODO(paul): register some bridge-wide standard ones here
+
+    // In case we're called after .run()
+    if (this.appService) {
+        metrics.addAppServicePath(this);
+    }
+
+    return metrics;
+};
+
+/**
+ * A convenient shortcut to calling registerBridgeGauges() on the
+ * PrometheusMetrics instance directly. This version will supply the value of
+ * the matrixGhosts field if the counter function did not return it, for
+ * convenience.
+ * @param {PrometheusMetrics~BridgeGaugesCallback} counterFunc A function that
+ * when invoked returns the current counts of various items in the bridge.
+ */
+Bridge.prototype.registerBridgeGauges = function(counterFunc) {
+    this.getPrometheusMetrics().registerBridgeGauges(() => {
```

Comment:
I don't like us not returning a `Promise` to let the caller know when this gauge has been registered. Please return it. Also, ES5 only (no fat arrow functions!).